components/atoms/Button/button.tsx

```diff
@@ -26,14 +26,12 @@ const Button = React.forwardRef<HTMLElement, ButtonsProps>(
         disabled ? "bg-light-orange-3 pointer-events-none text-light-orange-7 border-light-orange-5" : ""
       }`,
       link: `text-orange-600 hover:bg-orange-100 border-none ${disabled ? "text-orange-400 pointer-events-none" : ""}`,
+      destructive: "border-dark-red-8 bg-dark-red-8 text-white hover:border-dark-red-7 hover:bg-dark-red-7",
```

Comment:
This is the same styling we went with @isabensusan  in #1990. The styling got reverted at some point because it didn't match up with other delete buttons. A PR after this one will make all those delete buttons the `destructive` variant.